Client/Search Page/index.html

```diff
@@ -0,0 +1,178 @@
+    <p class="card-text"> Everybody loves noodles & pasta, whatever the type. Our noodle and pasta recipes include noodle soups, stir-fries, and tossed noodles, and Italian pastas.   </p>
+</div>      
+</div>
+<div class="search3">
+  <div class="card-body" style="width: 50rem;" >
+  <p class="card-text"> https://london.eater.com > maps > best-noodles-london</p>
+  <h5 class="card-title">Where to Find London's Best Noodles</h5>
+  <p class="card-text"> 1. Etles Uyghur Restaurant · 2. Dilara Uyghur Restaurant · 3. Xi'an Impression London · 4. Mama's Kubo · 5. Seto · 6. BunBunBun Vietnamese Food · 7. </p>
+</div>
+</div>
+</div>
+<div class="container-recipes">
+<div class="search4">
+  <div class="card-body" style="width: 50rem;" >
+    <p class="card-text"> https://thewoksoflife.com › Recipes
+       </p>
+    <h5 class="card-title">Noodles & Pasta: All Recipes - The Woks of Life</h5>
+    <p class="card-text"> Everybody loves noodles & pasta, whatever the type. Our noodle and pasta recipes include noodle soups, stir-fries, and tossed noodles, and Italian pastas.   </p>
+  </div>
+</div>
+
+<div class="search5"> 
+  <div class="card-body" style="width: 50rem;"  >
+
+  <p class="card-text"> https://thewoksoflife.com › Recipes
+     </p>
+  <h5 class="card-title">Noodles & Pasta: All Recipes - The Woks of Life</h5>
+  <p class="card-text"> Everybody loves noodles & pasta, whatever the type. Our noodle and pasta recipes include noodle soups, stir-fries, and tossed noodles, and Italian pastas.   </p>
+</div>
+</div>  
+<div class="search6">
+  <div class="card-body" style="width: 50rem;" >
+  <p class="card-text"> https://thewoksoflife.com › Recipes
+     </p>
+  <h5 class="card-title">Noodles & Pasta: All Recipes - The Woks of Life</h5>
+  <p class="card-text"> Everybody loves noodles & pasta, whatever the type. Our noodle and pasta recipes include noodle soups, stir-fries, and tossed noodles, and Italian pastas.   </p>
+</div>
+</div>
+<div class="search7">
+  <div class="card-body" style="width: 50rem;" >
+  <p class="card-text"> https://thewoksoflife.com › Recipes
+     </p>
+  <h5 class="card-title">Noodles & Pasta: All Recipes - The Woks of Life</h5>
+  <p class="card-text"> https://jamieoliver.com > Recipes > pasta-recipes
+     </p>
+  <h5 class="card-title">25 Best Ramen Noodles </h5>
+
+  <p class="card-text"> Everybody loves noodles & pasta, whatever the type. Our noodle and pasta recipes include noodle soups, stir-fries, and tossed noodles, and Italian pastas.   </p>
+</div>
+</div>  
+<div class="search8">
+  <div class="card-body" style="width: 50rem;" >
+  <p class="card-text"> https://thewoksoflife.com › Recipes
+     </p>
+  <h5 class="card-title">Noodles & Pasta: All Recipes - The Woks of Life</h5>
+  <p class="card-text"> Everybody loves noodles & pasta, whatever the type. Our noodle and pasta recipes include noodle soups, stir-fries, and tossed noodles, and Italian pastas.   </p>
+</div>
+</div>
+</div>
+</body>
+</html>
```

Comment:
Googlarachi in top left taking back to home is an excellent addition
Whole page looks incredible, like actually looking at google
the time taken addition is a great extra feature

one approach to make dynamic calls would be to contain all js in one file, use the storage to pass into fetch once you have been diverted to the search page